core/node/da_dispatcher/src/da_dispatcher.rs

```diff
@@ -27,178 +36,296 @@ impl DataAvailabilityDispatcher {
         config: DADispatcherConfig,
         client: Box<dyn DataAvailabilityClient>,
     ) -> Self {
+        let request_semaphore = Arc::new(tokio::sync::Semaphore::new(100));
```

Comment:
the value passed to the Semaphore constructor can be moved to a constant and documented .

Reply:
Same as above, it's now located in the config file.